eng/pipelines/dotnet-monitor-release.yml

```diff
@@ -163,8 +163,8 @@ stages:
           - powershell: Install-PackageProvider -Name NuGet -Force -Scope CurrentUser
             displayName: Install NuGet PowerShell Package Provider
 
-          - powershell: Install-Module Az.Storage -Force -Scope CurrentUser -AllowClobber -Verbose
-            displayName: Install Az.Storage Module
+          - powershell: Install-Module Az.Storage -Force -Scope CurrentUser -AllowClobber -Verbose -RequiredVersion 5.10.1
```

Comment:
Is the breaking change a on major / minor version bump? If so does `Install-Module` support wildcards here, e.g. `-RequiredVeresion 5.10.*`?

Given this is an interim solution I'm fine with it as-if if not.

Reply:
It doesn't seem to accept wildcards (`Cannot convert value '5.10.*' to type 'System.Version'`) and trying `5.10` will install `5.10.0`, so I think we're stuck with the exact version.